tests/unit/Rules/PHPLabelTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of Respect/Validation.
+ *
+ * (c) Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
+ *
+ * For the full copyright and license information, please view the "LICENSE.md"
+ * file that was distributed with this source code.
+ */
+
+namespace Respect\Validation\Rules;
+
+/**
+ * @group  rule
+ * @covers Respect\Validation\Rules\PHPLabel
+ * @covers Respect\Validation\Exceptions\PHPLabelException
+ */
+class PHPLabelTest extends \PHPUnit_Framework_TestCase
+{
+    protected $phpLabelValidator;
+
+    protected function setUp()
+    {
+        $this->phpLabelValidator = new PHPLabel();
+    }
+
+    /**
+     * @dataProvider providerForPHPLabel
+     */
+    public function testValidPHPLabelShouldReturnTrue($input)
+    {
+        $this->assertTrue($this->phpLabelValidator->__invoke($input));
+        $this->assertTrue($this->phpLabelValidator->assert($input));
+        $this->assertTrue($this->phpLabelValidator->check($input));
+    }
+
+    /**
+     * @dataProvider providerForNotPHPLabel
+     * @expectedException Respect\Validation\Exceptions\PHPLabelException
+     */
+    public function testInvalidPHPLabelShouldThrowPhoneException($input)
+    {
+        $this->assertFalse($this->phpLabelValidator->__invoke($input));
+        $this->assertFalse($this->phpLabelValidator->assert($input));
```

Comment:
I think it's not going to happen, see https://github.com/sebastianbergmann/phpunit/issues/1914#issuecomment-148407321